crt_portal/cts_forms/templates/forms/phone_pro_template.html

```diff
@@ -80,8 +93,44 @@ <h2>Contact</h2>
       {% include 'partials/end_expandable.html' %}
       {% endif %}
 
-      {% for additional_contact in additional_contacts %}
+      {% if section == "ELS-CRU" %}
+      {% include 'partials/end_expandable.html' %}
+      <hr>
+      {% include 'partials/start_expandable.html' with title='Respondent' %}
+      {% endif %}
+
+      {% if section != "ELS-CRU" %}
+      <h2>Incident location</h2>
+      {% endif %}
+
+      <div class="grid-row margin-top-2">
+        {% include 'partials/field.html' with field=form.location_name %}
+      </div>
+
+      <div class="grid-row margin-top-2">
+        {% include 'partials/field.html' with field=form.location_address_line_1 %}
+      </div>
+
+      <div class="grid-row margin-top-2">
+        {% include 'partials/field.html' with field=form.location_address_line_2 %}
+      </div>
+
+      <div class="grid-row margin-top-2">
+        {% include 'partials/field.html' with field=form.location_city_town field_size=6 extra_class="padding-right-2" %}
+        {% include 'partials/field.html' with field=form.location_state field_size=6 %}
+      </div>
+
+      <div class="grid-row margin-top-2 margin-bottom-3">
+        {% include 'partials/field.html' with field=form.location_zipcode %}
+      </div>
+
+      <hr>
 
+      {% if section == "ELS-CRU" %}
+      {% include 'partials/end_expandable.html' %}
+      {% endif %}
+
+      {% for additional_contact in additional_contacts %}
```

Comment:
should this go above, with the other if els-cru?

Reply:
I am not sure if it can, since prior to this is the incident_location fields. In the VOT form, these exist outside of an expandable, but in the ELS-CRU proform, these are inside the "Respondent" expandable.